docs/content/en/docs/yaml-crd-ref/evaluationdefinition.md

```diff
@@ -0,0 +1,99 @@
+---
+title: KeptnEvaluationDefinition
+description: Define an evaluation query
+weight: 20
+---
+
+A `KeptnEvaluationDefinition` defines evaluation tasks
+that can be run by the Keptn Lifecycle Toolkit
+as part of pre- and post-analysis phases of a workload or application.
+
+## Yaml Synopsis
+
+```yaml
+apiVersion: lifecycle.keptn.sh/v1alpha3
+kind: KeptnEvaluationDefinition
+metadata:
+  name: <evaluation-name>
+spec:
+  objectives:
+    - evaluationTarget: ">1"
+      keptnMetricRef:
+        name: available-cpus
+        namespace: some-namespace
+```
+
+## Fields
+
+* **apiVersion** -- API version being used.
+  Must be `v1alpha3` or later for this syntax.
```

Comment:
This will be painful to maintain. At each new API release, we would need to change it. I suggest creating a pointer towards the autogenerated docs of the CRDs to find the right version.